app/src/main/res/values-nl/strings.xml

```diff
@@ -54,6 +55,10 @@
     <string name="action_skip">Overslaan</string>
     <string name="action_restart_tutorial">Herstart Uitleg</string>
 
+    <!-- snackbar -->
+    <string name="snackbar_description">Weet u zeker dat u het wilt verwijderen?</string>
+    <string name="snackbar_undo">Ongd. maken</string>
```

Comment:
Change this to 'Zet Terug'

Reply:
Ik heb "Ongd. Maken" gezet omdat het ook bij gmail staat als je een mail weg haalt.